lib/routes/authorization.js

```diff
@@ -142,6 +146,9 @@ module.exports = {
         .valid(ACCESS_TYPE_OFFLINE, ACCESS_TYPE_ONLINE)
         .default(ACCESS_TYPE_ONLINE)
         .optional(),
+      // PKCE support
+      code_challenge_method: Joi.string().optional(), // Force 'S256'
```

Comment:
Note that the spec says this defaults to `plain` if unspecified, so I think we'll need to insist on it being present with a value of `S256`.